server/api/store.go

```diff
@@ -74,6 +76,10 @@ func newStoreInfo(store *metapb.Store, status *server.StoreStatus) *storeInfo {
 			Uptime:             typeutil.NewDuration(status.GetUptime()),
 		},
 	}
+	if time.Now().Sub(status.LastHeartbeatTS) > defaultStoreDownTime && store.State == metapb.StoreState_Up {
+		s.Store.StateName = "Down"
```

Comment:
I think we can construct a store and status, then call newStoreInfo and check the StateName. 

Now, I think we can define the const name in PD.